en/docs/basics_topics/distributed_train.md

```diff
@@ -1,35 +1,35 @@
 # Distributed training
 
-In OneFlow, you only need few simple configuration and the frame of OneFlow will automatically deal with calling job function, resources parallelism  and other issue. Thus, we do not need change network structure and logic of code. Then we can easily use distributed training.
+In OneFlow, you only need few simple lines of configuration and OneFlow will automatically deal with tasks scheduling, resources parallelism  and so on. Thus, we do not need change network structure and the logic of code. Then we can easily use distributed training.
 
-The ability of distributed training in OneFlow is very outstanding. This is the **main characters**distinguished between other framework.
+OneFlow's unique distributed training capability is the **most important feature** that sets OneFlow apart from other frameworks.
 
 In this article, we will introduce:
 
-* How to change a solo program to distributing program.
+* How to switch a program from single machine to distributed system.
 
-* The concept and job division of node in OneFlow.
+* The concept and job of node in OneFlow.
 
 ## The distribution advantage of OneFlow.
 
-* OneFlow use decentralized and flow framework. Not like  `master` and `worker` frame, it can maximum optimize the network speed between nodes.
+* OneFlow use decentralized streaming architecture. Not like  `master` and `worker` architecture, it can optimize the communication efficiency of node network to the maximum extent.
 
-* Support for  `consistent view`, the whole network only need only logic input and output.
+* Support for  `consistent view`, the whole network only need a logic input and output.
 
-* Also support to adapt with `mirrored view` from other framework. User who familiar with the distributed training in other frame can easily use OneFlow.
+* Provide a `mirrored view` compatible with other frameworks. Users who are familiar with the distributed training of other frameworks can start it quickly. 
```

Comment:
Provide a `mirrored view` compatible with other frameworks
->
A `mirrored view`  is provided compatible with other frameworks